cmd/brew-services.rb

```diff
@@ -304,7 +304,8 @@ def start(target, custom_plist = nil)
     def stop(target)
       if target.is_a?(Service) && !target.loaded?
         rm target.dest if target.dest.exist? # get rid of installed plist anyway, dude
-        odie "Service `#{target.name}` not running, wanna start it? Try `#{bin} start #{target.name}`"
+        odie "Service `#{target.name}` not started as `#{ServicesCli.root? ? "root" : ServicesCli.user}`, wanna start it? Try `#{bin} start #{target.name}`" +
+        ( target.started? ? "\nService `#{target.name}` is started as `#{target.started_as}` tho, wanna stop it? Try `#{ServicesCli.root? ? "" : "sudo "}#{bin} start #{target.name}`" : "" )
```

Comment:
I think it would be nice to have this be a single error message rather than a new line with a separate message.